src/react-components/avatar-selector.js

```diff
@@ -98,33 +98,15 @@ class AvatarSelector extends Component {
     });
   }
 
-  componentDidUpdate(prevProps) {
```

Comment:
Guessing we no longer need this because of the new animation component?

Reply:
Yup, it works without it now.

Reply:
Nvm. I misunderstood what this did. I added it back in and updated it for the new animation component.